### ISSUE TITLE
Error out of now where

### ISSUE THREAD

**Issue by Wolf68k** [MultiMC/MultiMC5#1067]
I got this earlier no idea what it means. All I know is that everything was going fine then about 20 error popups came out of no where.
I had an instance running but it didn't seem to be affected by this.
```
This is from the Event Log (Win7)
Faulting application name: MultiMC.exe, version: 1.0.0.0, time stamp: 0x00000000
Faulting module name: libstdc++-6.dll, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0x40000015
Fault offset: 0x0001f1db
Faulting process id: 0x1a2c
Faulting application start time: 0x01d0a2c793c5cb0e
Faulting application path: E:\Minecraft\MultiMC\MultiMC.exe
Faulting module path: E:\Minecraft\MultiMC\libstdc++-6.dll
Report Id: 92630d49-0efd-11e5-bc4d-001b213885de


Followed by:
Fault bucket 950900189, type 17
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: MultiMC.exe
P2: 1.0.0.0
P3: 00000000
P4: libstdc++-6.dll
P5: 0.0.0.0
P6: 00000000
P7: 40000015
P8: 0001f1db
P9: 
P10: 

Attached files:
C:\Users\Wolf\AppData\Local\Temp\WERBB09.tmp.WERInternalMetadata.xml

These files may be available here:
C:\Users\Wolf\AppData\Local\Microsoft\Windows\WER\ReportArchive\AppCrash_MultiMC.exe_0fb31ad74a615568bb97f12734f99b2f8f33b3_3bc82282

Analysis symbol: 
Rechecking for solution: 0
Report Id: 92630d49-0efd-11e5-bc4d-001b213885de
Report Status: 0
```
If you want the file, or what is in the file, mentioned above let me know and I'll add it.

**Comment by peterix** [comment]
Same as #1046 and #1008.

Everything I said there applies here too:
* Your Minecraft is logging enough to use up gigabytes of memory on log files. you should fix that.
* MultiMC doesn't handle or report this state yet. this should be fixed soon.

**Comment by Wolf68k** [comment]
Ok. Thank you for your time, sorry to have wasted it.
I did a search for the dll it listed but nothing came up and I wasn't sure what else to search for.

**Comment by peterix** [comment]
Fixed in develop: #1008
